galaxy_ng/app/api/ui/serializers/execution_environment.py

```diff
@@ -252,6 +252,7 @@ class ContainerRegistryRemoteSerializer(
     updated_at = serializers.DateTimeField(source='pulp_last_updated', required=False)
     last_sync_task = utils.RemoteSyncTaskField(source="*")
     write_only_fields = serializers.SerializerMethodField()
+    is_indexable = serializers.SerializerMethodField()
```

Comment:
I wonder if it would not be possible to use 
```suggestion
    is_indexable = serializers.BooleanField(source="get_registry_backend")
```


looks like source can be a callable https://www.django-rest-framework.org/api-guide/fields/#source

Will that work?